x-pack/plugins/apm/public/components/shared/transaction_action_menu/sections.ts

```diff
@@ -40,6 +42,8 @@ export const getSections = ({
   location: Location;
   urlParams: ApmUrlParams;
 }) => {
+  const apmRouter = useApmRouter();
```

Comment:
Not sure if we should use a hook in a non-hook?

Reply:
In this case it's safe since `getSections` is consistently called in the render function of the ActionMenuSections component. But I'll call it explicitly in the component render and pass it in. Alternative we could modify `getSections` to use the hook pattern as `useSections`.